MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -553,12 +553,18 @@ private async Task<string> ExtractImageInternal(string inputPath, string contain
 
             var mapArg = imageStreamIndex.HasValue ? (" -map 0:v:" + imageStreamIndex.Value.ToString(CultureInfo.InvariantCulture)) : string.Empty;
 
-            var enableThumbnail = !new List<string> { "wtv" }.Contains(container ?? string.Empty, StringComparer.OrdinalIgnoreCase);
             // Use ffmpeg to sample 100 (we can drop this if required using thumbnail=50 for 50 frames) frames and pick the best thumbnail. Have a fall back just in case.
-            var thumbnail = enableThumbnail ? ",thumbnail=24" : string.Empty;
+            var enableThumbnail = useIFrame && !new List<string> { "wtv" }.Contains(container ?? string.Empty, StringComparer.OrdinalIgnoreCase);
```

Comment:
```suggestion
            var enableThumbnail = useIFrame && !string.Equals("wtv", container, StringComparer.OrdinalIgnoreCase);
```
or am i reading this wrong?

Reply:
(useIFrame && container != "wtv") => enableThumbnail

Reply:
In which case, my suggested change is correct.